fl4health/strategies/flash.py

```diff
@@ -0,0 +1,164 @@
+from typing import Callable, Dict, List, Optional, Tuple, Union
+
+import numpy as np
+from flwr.common import (
+    FitRes,
+    MetricsAggregationFn,
+    NDArrays,
+    Parameters,
+    Scalar,
+    ndarrays_to_parameters,
+    parameters_to_ndarrays,
+)
+from flwr.server.client_proxy import ClientProxy
+from flwr.server.strategy import FedOpt
+
+
+class FLASH(FedOpt):
+    """FLASH - Fast and Robust Federated Learning Through Approximate Second-Order Method.
```

Comment:
Similar comment to the readme file above, the title doesn't seem to match that in the linked PDF

Reply:
corrected!